src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class SortCommand extends Command {
```

Comment:
Header comments for public class. https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#comments